iam/terraform/account-specific/main/lambda-logs-to-elasticsearch.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_lambda_function" "logs_to_es" {
+  filename = "../../../../aws/lambdas/LogsToElasticSearch_info.zip"
```

Comment:
I don’t see where this zip file is generated — do we need an `archive_file` resource?

Reply:
Ooh, I see that it’s committed — I would love to use `archive_file` to generate it instead and add the `zip` to `.gitignore` so we only have one copy of the code in the repo (the other lambdas use `archive_file` already to generate zips — ripe for inspiration!).

Reply:
Done! https://github.com/ustaxcourt/ef-cms/pull/387/commits/b6d2d97e80083e6e5196a332b1b35eaf76fd7955
https://github.com/ustaxcourt/ef-cms/pull/387/commits/6ed2048c0aabb444b7c5ad30260894293886f169
https://github.com/ustaxcourt/ef-cms/pull/387/commits/ae8b6b68eff79543291f1bb63cebcb3188873e95